Makefile.js

```diff
@@ -74,13 +74,53 @@ target.lint = function() {
     }
 };
 
+target.fixlint = function() {
```

Comment:
Is there a way we can avoid duplicating all of this logic from the lint target? It seems easy to forget to update one when updating the other.

Reply:
i can see two approaches in this to keep it consistent. 
- using `process.argv` to get the `fix` flag and run it in `target.lint` method itself.
- extract the logic from both in a separate function and simply pass a `fix` parament to the function call for `fixlint`

IMO, I would go with the first approach of using `process.argv`